docs/development/intro.rst

```diff
@@ -201,6 +202,13 @@ Integrate the code in Gammapy
 * check tests locally
 * check docs locally
 
+Contribute with Jupyter notebooks
+=================================
+
+* check tests with user tutorials environment
+* strip the output cells and code format
+* diff stripped notebooks
```

Comment:
We should give the commands to run here (and everywhere later), otherwise devs have the extra task of figuring out which exact command to run for each step. Of course, a text description of the task can remain if useful, but the most important thing is to give the commands needed.

Reply:
Ok.
Done.

I have just added the commands, more text is needed and will come in other PR together with the rest of the TODO sections remaining in the same page.